src/main/java/org/checkerframework/languageserver/ServerMain.java

```diff
@@ -16,11 +16,16 @@
 
 /** This class is the launcher of the language server. */
 public class ServerMain {
-
+    /** The logger for issuing information in the launcher of the language server. */
     private static final Logger logger = Logger.getLogger(ServerMain.class.getName());
 
+    /** The framework path and field is option in the client side. */
```

Comment:
What does the `is` mean here and below? Also the `client side` is not clear. What client is this talking about?

Reply:
> What does the `is` mean here and below? Also the `client side` is not clear. What client is this talking about?

Thanks, I will rewording the sentence.